python/ray/experimental/channel/torch_tensor_type.py

```diff
@@ -1,65 +1,99 @@
-from typing import TYPE_CHECKING, Any, List, Optional, Tuple
+from typing import TYPE_CHECKING, List, Optional, Tuple, Union
 
 import ray
-from ray.experimental.channel import ChannelOutputType
-from ray.util.annotations import DeveloperAPI, PublicAPI
+from ray.experimental.channel import ChannelContext, ChannelOutputType
+from ray.util.annotations import PublicAPI
 
 if TYPE_CHECKING:
-    import numpy as np
     import torch
 
-# Add this much padding to all shared memory buffers used to store tensors.
-# This shouldn't affect performance because the buffer is allocated once and
-# when transferring across nodes, only the serialized buffer is copied.
-TENSOR_BUFFER_PADDING_FRACTION = 0.2
-# 100KB so that we have room to store an exception if needed.
-MIN_TENSOR_BUFFER_SIZE = 100_000
+# 100KB to store metadata and/or exceptions.
+TENSOR_PADDING_SIZE_BYTES = 100_000
```

Comment:
QQ: isn't 100KB too big for this?

Reply:
There's not too much overhead other than CPU memory consumption here, so I think it's okay for now. For remote data, only the actual object should get transferred, not the entire allocated buffer.

We can make it adjustable in the future.